paystell-frontend/src/app/PaymentLinkScreen/page.tsx

```diff
@@ -1,132 +1,19 @@
-import * as Dialog from '@radix-ui/react-dialog';
-import Image from 'next/image';
-import Logo_PayStell from './Images/Logo_PayStell.png';
-import Logo_Company from './Images/Logo_Company.png';
-import Logo_PayStell_P from './Images/Logo_PayStell_P.png';
-import Product from './Images/Product.png';
-import { Button } from "@/components/ui/button";
+import { Header } from "./Components/header";
+import { ProductInfo } from "./Components/ProductInfo";
+import { PaymentSummary } from "./Components/PaymentSummary";
+import { PaymentButton } from "./Components/PaymentButton";
+import { Footer } from "./Components/footer";
```

Comment:
imports are updated accordingly
```suggestion
import { Header } from "@/components/PaymentLinkScreen/header";
import { ProductInfo } from "@/components/PaymentLinkScreen/ProductInfo";
import { PaymentSummary } from "@/components/PaymentLinkScreen/PaymentSummary";
import { PaymentButton } from "@/components/PaymentLinkScreen/PaymentButton";
import { Footer } from "@/components/PaymentLinkScreen/footer";
```

Reply:
if I move the Component folder to "src/components/PaymentLinkScreen" the imports will be  "import { Header } from "@/components/PaymentLinkScreen/Components/header" unless you want me to remove the components from the "Components" folder